### ISSUE TITLE
❓[Question] The training details of the first stage class-agnoistic mask proposal network.

### ISSUE THREAD

**Issue by sennnnn** [mlpc-ucsd/MaskCLIP#3]

**Comment by zh-ding** [comment]
We use the Mask2Former/MaskRCNN as our first-stage class-agnostic mask proposal network by removing its classification head. Other training details e.g., learning rate, iterations stay the same as the original method.

**Comment by sennnnn** [comment]
Thanks for your response. I'm curious about the training settings of MaskCLIP for open-vocabulary instance segmentation. Could you provide the corresponding implementation?

**Comment by clownrat6** [comment]
Thanks for your response 🙏
